nexus/tests/output/nexus_tags.txt

```diff
@@ -132,6 +132,8 @@ session_sshkey_view                      /session/me/sshkeys/{ssh_key_name}
 
 API operations found with tag "silos"
 OPERATION ID                             URL PATH
+policy_update                            /policy
+policy_view                              /policy
 silo_create                              /silos
 silo_delete                              /silos/{silo_name}
 silo_identity_provider_create            /silos/{silo_name}/saml-identity-providers
```

Comment:
I'm not certain this tag grouping is really correct. I'm not recommending a change in this PR, but broadly it's worth us talking about what we're going to call "global" in the context of the silo. If we're going to use the tag silo (and thus silo as a term to describe this global context) then we're going to need to educate users on it even before they really need to know what a silo is.